### ISSUE TITLE
Webpack Internal Typings

### ISSUE THREAD

**Issue by dl748** [DefinitelyTyped/DefinitelyTyped#49757]
Webpack contains its own typings in the form of

https://github.com/webpack/webpack/blob/master/types.d.ts

The README states

```
Removing a package
When a package bundles its own types, types should be removed from Definitely Typed to avoid confusion.

You can remove it by running npm run not-needed -- typingsPackageName asOfVersion [libraryName].
```

Unfortunately, removing @types/webpack as of 5.0.0 cause many other typings to fail.

So I'm wondering what is the correct way in this situation.  I created a Draft PR for this removal.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49755

I opened this issue as a discussion on what is the best approach for updating these typings.

**Comment by nzjony** [comment]
I have the same issue with @types/webpack-merge, see: https://github.com/survivejs/webpack-merge/issues/160

**Comment by Stanzilla** [comment]
Yeah @types/copy-webpack-plugin also depends on @types/webpack which causes it to fail.

**Comment by rtfpessoa** [comment]
After some tries I got a temporary way around it by resolving the types to the actual webpack package.
It is quite a hack but works for now. If you have any better suggestions let me know :pray: 
```
  "resolutions": {
    "@types/mini-css-extract-plugin/@types/webpack": "https://registry.yarnpkg.com/webpack/-/webpack-5.23.0.tgz#9ed57e9a54b267b3549899271ad780cddc6ee316",
    "@types/copy-webpack-plugin/@types/webpack": "https://registry.yarnpkg.com/webpack/-/webpack-5.23.0.tgz#9ed57e9a54b267b3549899271ad780cddc6ee316"
  },
```
